src/Utility/ExprFunctions.C

```diff
@@ -186,6 +186,12 @@ EvalFunction::EvalFunction (const char* function)
   // Note, this will also catch things like tan(x), but...
   std::string expr(function);
   IAmConstant = expr.find_first_of('t') > expr.size();
+
+  // Need to do this, since they are fixed-size arrays, not vectors
+  for (auto& it : gradient)
```

Comment:
you can use the init list instead, gradient{}, dgradient{} or you can use class initialization, gradient = {}, dgradient = {}.